boards/arm/pinnacle_100_dvk/doc/index.rst

```diff
@@ -208,15 +208,6 @@ You can build and flash the examples to make sure Zephyr is running correctly on
 your board. The button and LED definitions can be found in
 :zephyr_file:`boards/arm/pinnacle_100_dvk/pinnacle_100_dvk.dts`.
 
-
-Overlay file naming
-===================
-
-The file has to be named ``<board>.overlay`` and placed in the app main directory to be
-picked up automatically by the build system.
-
-See :ref:`set-devicetree-overlays` for further details.
```

Comment:
This line was retained in the other files.

Reply:
But I guess in context this wasn't providing instructions on overlays, so it doesn't belong here.

Reply:
Yep, that was my thinking.